lms/static/scripts/frontend_apps/components/dashboard/DashboardActivityFilters.tsx

```diff
@@ -6,12 +6,12 @@ import { useConfig } from '../../config';
 import { useAPIFetch } from '../../utils/api';
 
 export type DashboardActivityFiltersProps = {
-  selectedCourses: Course[];
-  onCoursesChange: (newCourses: Course[]) => void;
-  selectedAssignments: Assignment[];
-  onAssignmentsChange: (newAssignments: Assignment[]) => void;
-  selectedStudents: Student[];
-  onStudentsChange: (newStudents: Student[]) => void;
+  selectedCourses: string[];
+  onCoursesChange: (newCourses: string[]) => void;
+  selectedAssignments: string[];
+  onAssignmentsChange: (newAssignments: string[]) => void;
+  selectedStudents: string[];
+  onStudentsChange: (newStudents: string[]) => void;
```

Comment:
Considering these are now IDs, perhaps we should rename them to `selectedCourseIds`, `selectedAssignmentIds` and `selectedStudentIds`.

Reply:
Yes, I think it would be helpful to add "Ids" here to clarify, as otherwise a reader might guess that `selectedStudents` is a list of student names for example.